cheri_compressed_cap_128m.h

```diff
@@ -84,16 +84,17 @@ __extension__ typedef signed __int128 cc128m_offset_t;
 typedef uint64_t cc128m_addr_t;
 typedef int64_t cc128m_saddr_t;
 #include "cheri_compressed_cap_macros.h"
+// Mode not stored inside capabilities, just define it as a simple enum matching the C64 bit in PSTATE
+typedef enum _CC_N(Mode) { _CC_N(MODE_CAP) = 1, _CC_N(MODE_INT) = 0 } _CC_N(Mode);
```

Comment:
Hm, I'd rather not have the dummy type, it's misleading to have it exist for a capability format that doesn't have it... AFAICT it's only used by the new make_max_perms_cap_ext Morello?

Reply:
Yeah that's the only use. I initially hoped I could get away with not defining it or using `typedef bool _CC_N(Mode)`, but this was simpler. Maybe we just don't provide the new API for Morello?

I guess we could provide the {get,set}_execution_mode APIs for Morello and just have them update the LSB of the address, but not sure that's too useful.

Reply:
Removed this now and just not providing the API instead.